allennlp/models/simple_tagger.py

```diff
@@ -127,7 +133,7 @@ def tag(self, text_field: TextField) -> Dict[str, Any]:
         # TODO(Mark): Generalise how the array is transformed into a variable after settling the data API.
         # Add a batch dimension by unsqueezing, because pytorch
         # doesn't support inputs without one.
-        array_input = torch.autograd.Variable(torch.LongTensor(array_input[0])).unsqueeze(0)
+        array_input = {"tokens": torch.autograd.Variable(torch.LongTensor(array_input["tokens"])).unsqueeze(0)}
```

Comment:
Hmm, we should have a method somewhere that produces variables from `DataArrays`.  If the array is a dict, we just convert all of the keys.